### PR TITLE
Clementine latest

### PR DESCRIPTION
**Pull request by narze**
This version differs from clementine-rc as it is using the latest build within https://builds.clementine-player.org/

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-versions/pulls
[closed issues]: https://github.com/caskroom/homebrew-versions/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/clementine-latest.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+cask 'clementine-latest' do
+  version '1.3.1-388'
+  sha256 'd34a9ab5c71c3e84420d6ce7e2f7426455a125a3a9aa86b86806188b2ecc0eac'
+
+  url 'https://builds.clementine-player.org/mac/clementine-1.3.1-388-gf1b767f.dmg'
```
Comment (commitay) [inline Casks/clementine-latest.rb]: Please change this to a [`url do`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#using-a-block-to-defer-code-execution) block.

`version` and `sha256` will need to be changed to `latest` and `no_check`

File: Casks/clementine-latest.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  name 'Clementine'
+  homepage 'https://www.clementine-player.org/'
+
+  conflicts_with cask: 'caskroom/cask/clementine'
```
Comment (commitay) [inline Casks/clementine-latest.rb]: [
'clementine',
'clementine-rc'
]

File: Casks/clementine-latest.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  preflight do
+    set_permissions "#{staged_path}/Clementine.app", '0755'
+  end
+
+  postflight do
+    set_permissions "#{appdir}/Clementine.app", '0555'
+  end
+
+  zap delete: [
```
Comment (commitay) [inline Casks/clementine-latest.rb]: Please update the `zap`

https://github.com/caskroom/homebrew-versions/blob/001992c33b299d68a504aeb8361c7573a21fe93c/Casks/clementine-rc.rb#L16-L23

**Comment by narze** [comment]
All done!
Also removed `postflight` due to this error on zapping
```
Error: Directory not empty @ dir_s_rmdir - /Applications/Clementine.app
```

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**